src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -188,9 +310,20 @@ const BaseCell: React.FC<React.HTMLAttributes<any>> = ({
   )
 }
 
-export const ImageCell: React.FC<React.HTMLAttributes<any>> = ({
-  children
+interface ImageCellProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode
+  isWireFrame?: boolean
+}
+
+export const ImageCell: React.FC<ImageCellProps> = ({
+  children,
+  isWireFrame = false
 }) => {
+  const wireFrameStyles = {
+    width: '118px',
```

Comment:
these numbers feel a bit magic, where do they come from, can we get anything from the theme?

Reply:
Turned them into percentages according to the design